app/modules/transcribe/modals/transcribe-next.controller.js

```diff
@@ -42,4 +43,32 @@ function TranscribeNextController($modalInstance, ClassificationFactory) {
             .then(close, error);
     }
 
+    function viewAnnotations() {
+        if (annotations.length > 0) {
+            var i, doc = new jsPDF('p', 'mm');
+            for (i = 0; i < annotations.length; i++) {
```

Comment:
Rather than use a `for` loop, you can use `forEach`.

Also, why are you iterating over the annotations with the loop and the `map()` below?